cclibs/features/short-article-promotion.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+export default function decoratePromotion(promotionElement) {
```

Comment:
```suggestion
export default function init(el) {
```

I prefer much more concise naming conventions.
1. **decoratePromotion --> init** - Why? The file is already called `short-article-promotion.js` It's implied in the file of what you are doing. Especially since this is a default export.
2. **promotionElement --> el** - Why? It's not a promotion yet, you are feeding in a raw element that has not been decorated.

When you use the above pattern, you get out of having to think about names, and it's much easier to copy paste to other blocks.

1. What do we want? (name of file)
2. What are we doing? (name of function)
3. What are we doing it to? (raw element)